### ISSUE TITLE
Using PDO [instead of mysql_connect()] in the SQLWorker.php example

### ISSUE THREAD

**Issue by andrewspy** [krakjoe/pthreads#120]
Hi,

Anyway to make PDO workable in SQLWorker.php example? When I try to connect in the worker, it just throw Exception that PDO cannot be serialize/unserialize.

> PHP Fatal error:  Uncaught exception 'PDOException' with message 'You cannot serialize or unserialize PDO instances' in ...

``` php
class MyWork extends Stackable{
    function run(){
        $pdo  = $this->worker->get_connection();

        // Do something with the connection
    }
}

class MyWorker extends Worker{
    public $pdo;

    function __construct($conf){
        $this->conf = $conf;
    }

    function run(){
        $this->pdo = new PDO('mysql:host=localhost;dbname=test');
        //$this->pdo = mysql_connect('localhost', 'root', '');
    }

    function get_connection(){
        return $this->pdo;
    }
}

$worker = new MyWorker(array(
    'url' => 'http://localhost',
));
$worker->start();
$worker->stack(new MyWork);
```

Is PDO an object or a resource?

Thanks!

**Comment by NaibStilgar** [comment]
Hi, that is a nice example. I have also the case where I would like to connect from both parent & child. I want to use that for logging purpose through log4php. I have no issue for files, but it fails with PDO.

**Comment by krakjoe** [comment]
This is not a place that is intended to ask and answer questions.

Stackoverflow is, and I hang around there and try to answer anything related to pthreads when I have the time to browse questions and write example code.

Actual bugs, and development, have priority over helping people, but that doesn't mean I'm unwilling to help.

Conducting these kinds of conversations on Stackoverflow exposes them to many more people than conducting them here, it won't be that long before you don't have to wait for me to answer questions any more, because knowledge is dispersed more effectively.

TL;DR please don't ask for help here, it's not the right place ...
